magento/devdocs#: Layout instructions. Improvement - adding changes to Configure services as web APIs topic (#6905)
magento/devdocs: src/guides/v2.3/extension-dev-guide/service-contracts/service-to-web-service.md

You can configure a Magento or third-party service as a web [API](https://glossary.magento.com/api).

> To [configure a web API](#configure-webapi), you define [XML](https://glossary.magento.com/xml) elements and attributes in the `webapi.xml` XML configuration file for the [module](https://glossary.magento.com/module) for the service. The `webapi.xml` file for your module specifies an XML schema file for validation. By default, this file is `app/code/Magento/Webapi/etc/webapi.xsd` or `vendor/magento/module-webapi/etc/webapi.xsd`.

Reviewer: I would suggest adjusting this part to clarify all statements

```suggestion
To [configure a web API](#configure-webapi), you define [XML](https://glossary.magento.com/xml) elements and attributes in the `webapi.xml` XML configuration file for the [module](https://glossary.magento.com/module) for the service. The `etc/webapi.xsd` file for your module specifies an XML schema file for validation. The default XML schema validation rules are stored in `app/code/Magento/Webapi/etc/webapi.xsd` or `vendor/magento/module-webapi/etc/webapi.xsd` file.
```

Thank you
